README.md

```diff
@@ -57,6 +57,15 @@ You do not have to create a dedicated token. Make sure to use the GitHub's defau
 **Optional** By default ZAP Docker container will fail with an [exit code](https://github.com/zaproxy/zaproxy/blob/7abbd57f6894c2abf4f1ed00fb95e99c34ef2e28/docker/zap-api-scan.py#L35),
 if it identifies any alerts. Set this option to `true` if you want to fail the status of the GitHub Scan if ZAP identifies any alerts during the scan.
 
+## Environment variables
+
+If set, the following [ZAP authentication environment variables](https://www.zaproxy.org/docs/authentication/handling-auth-yourself/#authentication-env-vars)
+will be copied into the docker container:
+
+- `ZAP_AUTH_HEADER_VALUE` - if this is defined then its value will be added as a header to all of the requests
+- `ZAP_AUTH_HEADER` - if this is defined then its value will be used as the header name - if it is not defined then the standard Authorization header will be used
+- `ZAP_AUTH_HEADER_SITE` - if this is defined then header will only be included in sites who’s name includes its value
```

Comment:
IMO it could just link, less duplication/maintenance.

Reply:
Keep the names and drop the descriptions? I think it's important to document precisely which env vars are recognized (or else use an env file instead of `-e`, but that's less convenient for the user).

Reply:
Sounds good.